docs/src/main/asciidoc/logging.adoc

```diff
@@ -56,24 +51,22 @@ public class ExampleResource {
 ----
 
 NOTE: While JBoss Logging routes log messages into JBoss Log Manager directly, one of your libraries might rely on a different logging API.
-In such cases, you need to use a <<logging-adapters,logging adapter>> to ensure that its log messages are routed to JBoss Log Manager as well.
-
+In such cases, you need to use a <<logging-apis, logging adapters>> to ensure that its log messages are routed to JBoss Log Manager as well.
 
-== Methods of obtaining an application logger
+== Get an application logger
 
 In Quarkus, the most common ways to obtain an application logger are by:
 
 * <<declaring-a-loger-field,Declaring a logger field>>
 * <<logging-with-panache,Logging with Panache>>
 * <<injection-of-a-configured-logger,Injecting a configured logger>>
 
-
 [[declaring-a-loger-field]]
 === Declaring a logger field
 
 With this classic approach, you use a specific API to obtain a logger instance, store it in a static field of a class, and call logging operations upon this instance.
 
-The same flow can be applied with any of the <<supported-logging-apis,supported logging APIs>>.
+The same flow can be applied with any of the <<logging-apis, supported logging APIs>>.
```

Comment:
Is this space okay? We don't use a space here in most other places, it would seem.

Reply:
Yes, the asciidoc parser handles the space.